collect_app/src/main/res/values/strings.xml

```diff
@@ -85,6 +85,7 @@
     <string name="get_location">Record Location</string>
     <string name="provider_disabled_error">Sorry, Location providers are disabled!</string>
     <string name="invalid_answer_error">Sorry, this response is invalid!</string>
+    <string name="invalid_space_in_answer">Invalid Answer: The value \"%s\" should not have any spaces</string>
```

Comment:
I think this is going to be confusing to users because they don't know about underlying values and it's not clear how they can recover from it. Maybe "Warning: underlying value `\"%s\" has spaces"?

Reply:
ok